sympy/solvers/ode/single.py

```diff
@@ -193,6 +194,9 @@ def matches(self) -> bool:
             self._matched = self._matches()
         return self._matched
 
+    def _verify(self) -> bool:
+        return True
+
```

Comment:
I think it makes more sense for this to be a method of `SinglePatternODESolver`.